env.d/development/common.dist

```diff
@@ -27,6 +27,19 @@ JOANIE_ENROLLMENT_GRADE_CACHE_TTL=600
 # Payment Backend
 JOANIE_PAYMENT_BACKEND=joanie.payment.backends.dummy.DummyPaymentBackend
 
+## payplug payment backend settings
+# JOANIE_PAYMENT_BACKEND="joanie.payment.backends.payplug.PayplugBackend"
+# JOANIE_PAYMENT_CONFIGURATION={"secret_key": "fake_secret_key", "public_key": "fake_public_key"}
+
+## Lyra payment backend settings
+# LYRA_USERNAME="69876357"
+# LYRA_PASSWORD="testpassword_DEMOPRIVATEKEY23G4475zXZQ2UA5x7M"
+# LYRA_PUBLIC_KEY="69876357:testpublickey_DEMOPUBLICKEY95me92597fd28tGD4r5"
```

Comment:
Is that fake ? Otherwise we should not commit that nope ?

Reply:
No fake, but public lyra accesses available in their doc.